homeassistant/components/concord232/alarm_control_panel.py

```diff
@@ -19,39 +19,46 @@
 DEFAULT_HOST = 'localhost'
 DEFAULT_NAME = 'CONCORD232'
 DEFAULT_PORT = 5007
+DEFAULT_MODE = 'audible'
 
 SCAN_INTERVAL = datetime.timedelta(seconds=10)
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_CODE, 'code validation'): cv.string,
```

Comment:
This should be a const, too.

Reply:
It is if defined in configuration.yaml. If not it will be None. I was trying to make it as much as the Manual Alarm Panel as possible:
https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/manual/alarm_control_panel.py
see ln 77
is that in error too?

Are you suggesting adding:
```suggestion
    DEFAULT_CODE = None
    vol.Optional(CONF_CODE, default=DEFAULT_CODE): cv.string,
```

How would I go about removing the new feature fro the PR? Just recommit with the original file without changes?

Reply:
You are mixing things up here. The string "code validation" at alarm_control_panel.py is the identifier between the different vol.Exclusive options. What about:

```
vol.Optional(CONF_CODE): cv.string,
```

Reply:
Yes. You can recommit the initial file and check the diff here afterwards.

Reply:
ok!! Thanks!

Reply:
I have made a commit with the original file and updated the comment to only address the fix. Does it look ok?